src/NzbDrone.Core/Notifications/Trakt/Resource/TraktListResource.cs

```diff
@@ -0,0 +1,24 @@
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Notifications.Trakt.Resource
+{
+    public class TraktListResource
+    {
+        public int? Rank { get; set; }
+        [JsonProperty(PropertyName = "listed_at")]
+        public string ListedAt { get; set; }
+        public string Type { get; set; }
+
+        public int? Watchers { get; set; }
+
+        public long? Revenue { get; set; }
+        [JsonProperty(PropertyName = "watcher_count")]
+        public long? WatcherCount { get; set; }
+        [JsonProperty(PropertyName = "play_count")]
+        public long? PlayCount { get; set; }
+        [JsonProperty(PropertyName = "collected_count")]
+        public long? CollectedCount { get; set; }
+
+        public TraktEpisodeResource Movie { get; set; }
```

Comment:
👍 on the new line here, unless you want to group it with the other props. Need to rename `Movie` to `Episode`.